report.md

```diff
@@ -64,6 +61,104 @@ Plan for refactoring complex code:
 
 Estimated impact of refactoring (lower CC, but other drawbacks?).
 
+Method InlineDelegateByteBuddyMockMaker::InlineDelegateByteBuddyMockMaker():
+
+- CCN: 24
+
+- Possible refactor: a large part at the beginning (lines 263 to 265) can be extracted into a separate function. This code deals with initialization error and is independent from the object creation. This part greatly lowers the CC yet has no real drawback. With only this refactor, the function is still above 15 CCN according to lizard's computation method. If needed, we can also refactor the lambda method to assign `isMockConstruction`  (lines 293 to 337) into a separate method.
```

Comment:
'onConstruction' (lines 293-237) could also be refactored out into a separate function.

Reply:
See answer to previous comment.